lib/open_project/gitolite/grack/git.rb

```diff
@@ -0,0 +1,128 @@
+require 'grack/git'
```

Comment:
This will make grack a fixed dependency of the plugin, which is not acceptible for some of our clients. I have no problem with grack being part of the plugin, but it needs to be configurable. Test whether `require 'grack'` succeeds and/or make a configuration setting to disable grack-based checkout.

We use https through our Apache module and thus will not maintain this feature. If you're unable to make this optional, I suggest to extract that part as an additional plugin.